tests/e2e/specs/dashboard/screens.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Dashboard screens test.
```

Comment:
The tests under `/dashboard/` here are more specific to the SK dashboard where as these tests are a bit closer to general wp-admin tests.

Let's move these to `specs/wp-admin/` accordingly 👍

Reply:
Thanks @aaemnnosttv this has been resolved in a recent commit.